spec.bs

```diff
@@ -119,11 +121,22 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
       text: face validation anchor; url: face-validation-anchor
     urlPrefix: webappapis.html
       text: fire a click event; url: fire-a-click-event
+      text: target browsing context; url:concept-environment-target-browsing-context
+spec: fetch; urlPrefix: https://fetch.spec.whatwg.org/
+  type: dfn
+    text: queue a cross-origin embedder policy CORP violation report; url: queue-a-cross-origin-embedder-policy-corp-violation-report
 spec: RFC8941; urlPrefix: https://www.rfc-editor.org/rfc/rfc8941.html
   type: dfn
     text: structured header; url: #section-1
     for: structured header
       text: token; url: name-tokens
+spec: LNA; urlPrefix: https://wicg.github.io/local-network-access/
+  type: dfn
+    text: Integration with HTML; url: integration-html
+    text: IP address space; url: policy-container-ip-address-space
+    for: response
+      text: IP address space; url: response-ip-address-space
```

Comment:
Is exported https://github.com/WICG/local-network-access/blob/main/index.src.html#L632